### PR TITLE
Add github PR and issue templates for Uno.Core

### PR DESCRIPTION
**Pull request by agneszitte**
Add github PR and issue templates for Uno.Core

### CODE REVIEW

**Review by jeremiethibeault — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## I'm submitting a...
+<!-- Please uncomment one or more that apply to this issue -->
+
+<!-- - Regression (a behavior that used to work and stopped working in a new release) -->
+<!-- - Bug report (I searched for similar issues and did not find one) -->  
+<!-- - Feature request --> 
+<!-- - Sample app request -->
+<!-- - Documentation issue or request -->
+<!-- - Question of Support request => Please do not submit support request here, instead see https://github.com/nventive/Uno/blob/master/README.md#have-questions-feature-requests-issues -->
```
Comment (MatFillion) [inline .github/ISSUE_TEMPLATE.md:14]: the link points to uno, do we have an equivalent in uno.core?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tested code with current [supported SDKs](../README.md#supported)
+- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
+- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
+- [ ] Contains **NO** breaking changes
+- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
```
Comment (MatFillion) [inline .github/PULL_REQUEST_TEMPLATE.md:33]: link to uno.core release notes :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## What is the new behavior?
+<!-- Please describe the new behavior after your modifications. -->
+
+
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tested code with current [supported SDKs](../README.md#supported)
+- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
```
Comment (MatFillion) [inline .github/PULL_REQUEST_TEMPLATE.md:30]: docs here point to uno. I think we can remove this checkbox line altogether, unless we have the equivalent in uno.core

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (agneszitte) [reply .github/ISSUE_TEMPLATE.md:14]: I will check

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (agneszitte) [reply .github/PULL_REQUEST_TEMPLATE.md:33]: I will update it, thanks

**Pull request closed without merging**